plugin.video.filmarkivet/changelog.txt

```diff
@@ -0,0 +1,7 @@
+v3.1.0  (2019-12-29)
```

Comment:
drop from submission if possible, this is no longer parsed by kodi

Reply:
The whole changelog you mean?